calculator/src/App.css

```diff
@@ -0,0 +1,52 @@
+.App-header {
```

Comment:
Minor point: usually CSS classes start with a lowercase letter. Or if you want uppercase then do uppercase consistently everywhere in the stylesheet (e.g. `.Activities` instead of `.activities` down below).